python/ray/serve/controller.py

```diff
@@ -263,9 +263,22 @@ async def deploy(self, name: str, backend_config: BackendConfig,
             goal_id = self.backend_state.create_backend(
                 name, backend_config, replica_config, version)
 
-            self.endpoint_state.create_endpoint(name, f"/{name}",
-                                                ["GET", "POST"],
-                                                TrafficPolicy({
-                                                    name: 1.0
-                                                }))
+            route = f"/{name}"
+            methods = ["GET", "POST"]
+            if replica_config.is_asgi_app:
+                route += "/{wildcard:path}"
```

Comment:
Hmm I'm still not understanding this logic. Should we just need to configure the base path in FastAPI to `/{name}`?